subjects/sliceremove/README.md

```diff
@@ -0,0 +1,41 @@
+## sliceremove
+
+### Instructions
+
+Write a function that takes a slice of integers and an `int` as arguments, if the given `int` exists in the slice then remove it, otherwise return the slice.
+- If the slice is empty, return the slice itself.
+
+### Expected function
+
+```go
+func SliceRemove(slice []int , num int) []int {
+
+}
+```
+
+### Usage
+
+Here is a possible program to test your function:
+
+```go
+package main
+
+import (
+	"fmt"
+
+	"piscine"
+)
+
+func main() {
+	fmt.Println(piscine.SliceRemove([]int{1, 2, 3}, 2))
+	fmt.Println(piscine.SliceRemove([]int{4, 3}, 4))
+}
```

Comment:
This is only a suggestion, maybe we can add one more output that matches this condition: "If the slice is empty, return the slice itself."

Reply:
fixed